office365.json

```
@@ -7528,6 +7528,102 @@
             ],
             "versions": "EQ(*)"
         },
+        {
+            "action": "report message",
+            "identifier": "report_message",
+            "description": "Add the sender email into the report",
+            "verbose": "This action takes as input an email whose sender will be added to the junk/notJunk/unknown/phish/unknownFutureValue Senders List. The message ID changes after the execution and is a required parameter for request hence undo action would require unique ID. Note that a message from the email address must exist in the user's mailbox before you can add the email address to or remove it from the Blocked Senders List.<ul><li>If the <b>is_message_move_requested</b> parameter is set to True, the sender of the target email message is added to the blocked sender list and the email message is moved to the junk/notJunk/unknown/phish/unknownFutureValue.</li><li>If the <b>is_message_move_requested</b> attribute is set to False, the sender of the target email message is added to the blocked sender list and the email message is not moved from the folder.</li></ul>To view the current Block Senders list, please read the following Powershell articles: <ul><li>https://docs.microsoft.com/en-us/powershell/exchange/exchange-online/connect-to-exchange-online-powershell/connect-to-exchange-online-powershell?view=exchange-ps</li><li>https://docs.microsoft.com/en-us/powershell/module/exchange/antispam-antimalware/Get-MailboxJunkEmailConfiguration?view=exchange-ps.</li></ul>",
+            "type": "contain",
+            "read_only": false,
+            "parameters": {
+                "message_id": {
+                    "description": "Message ID to pick the sender of",
+                    "data_type": "string",
+                    "required": true,
+                    "order": 0
+                },
+                "user_id": {
+                    "description": "User ID to base the action of",
+                    "data_type": "string",
+                    "required": true,
+                    "order": 1
+                },
+                "is_message_move_requested": {
+                    "description": "Indicates whether the message should be moved out of current folder",
+                    "data_type": "boolean",
+                    "order": 2
+                }
+                ,
```

looks like this comma formatting is causing pre commit to fail. Please remove the endline. 
```suggestion
                    "description": "Indicates whether the message should be moved out of current folder",
                    "data_type": "boolean",
                    "order": 2
                },
```